moped-editor/src/views/projects/projectView/ProjectComponents/utils/useUpdateComponent.js

```diff
@@ -233,7 +226,6 @@ export const useUpdateComponent = ({
   const [editState, editDispatch] = useReducer(editReducer, {
     isEditingComponent: false,
     showEditAttributesDialog: false,
-    showEditModeDialog: false,
```

Comment:
This piece of state isn't needed since it tracked whether the removed modal was displayed or not.